src/OnboardingSPA/components/ImageUploader/index.js

```diff
@@ -51,7 +51,7 @@ const ImageUploader = ({ icon, iconSetter }) => {
     function loader(){
         return (
         <div className="image-uploader_window">
-                <Loader/>
+                <ImageUploadLoader />
```

Comment:
The only place this is being used is BasicInfo, let's just shift the whole thing to **Loaders/ImageUpload** ??